Fatal error when installing zephyr with Visual Studio in Linux

### Describe the bug

I have run through all the steps in the [getting started guide](https://docs.zephyrproject.org/latest/develop/getting_started/index.html#next-steps) without issue, but am having issues with the [steps for visual studio](https://docs.zephyrproject.org/latest/develop/tools/vscode.html). When I try to run 
`west build -p always -b native_sim/native/64`
I get the error:
```-- west build: generating a build system

CMake Error at CMakeLists.txt:15 (message):
  A user error has occurred.

  cmake was invoked with '/home/NAME/zephyrproject/zephyr' specified as the
  source directory,

  but it must be invoked with an application source directory,

  such as '/home/NAME/zephyrproject/zephyr/samples/hello_world'.

  Debug variables:

  CMAKE_CACHEFILE_DIR: /home/NAME/zephyrproject/zephyr/build

-- Configuring incomplete, errors occurred!
See also "/home/NAME/zephyrproject/zephyr/build/CMakeFiles/CMakeOutput.log".
FATAL ERROR: command exited with status 1: /usr/bin/cmake -DWEST_PYTHON=/home/NAME/zephyrproject/.venv/bin/python3 -B/home/NAME/zephyrproject/zephyr/build -GNinja -DBOARD=native_sim/native/64 -S/home/NAME/zephyrproject/zephyr
```


### Regression

- [ ] This is a regression.

### Steps to reproduce

_No response_

### Relevant log output

```shell

```

### Impact

Showstopper – Prevents release or major functionality; system unusable.

### Environment

- Linux Mint

### Additional Context

_No response_

--- a/doc/develop/tools/vscode.rst
+++ b/doc/develop/tools/vscode.rst
@@ -52,7 +52,7 @@ required information (ex. include paths). You may do it from VS Code embedded te
 .. code-block:: console
 
    $ cd zephyr
-   $ west build -p always -b native_sim/native/64
+   $ west build -p always -b native_sim/native/64 samples/blinky
 
 
 Configure the C/C++ extension

